docs/index.md

```diff
@@ -67,11 +39,19 @@ email, or any other method with the owners of this repository before making a ch
 {% endfor %}
 </ul>
 
-### Code of Conduct
+### Background on technology
+
+This site uses [Just the Docs] for its template.
 
-Just the Docs is committed to fostering a welcoming community.
+If [Jekyll] is installed on your computer, you can build and preview the created site *locally*. This lets you test changes before committing them, and avoids waiting for GitHub Pages.[^2] And you will be able to deploy your local build to a different platform than GitHub Pages.
 
-[View our Code of Conduct](https://github.com/just-the-docs/just-the-docs/tree/main/CODE_OF_CONDUCT.md) on our GitHub repository.
+More specifically, the created site:
```

Comment:
I think we should remove this two lines (not the note), lines 48-50

You can see a preview below 
![image](https://user-images.githubusercontent.com/29259906/207659102-55b1011d-9de3-49fe-ac69-ec8a725843f9.png)

Reply:
ok! as a contributor who forgets these details I find them handy sometimes, but I will just accept that I may not be able to keep up with the technologies. I don't make changes often enough to have the configuration maintained and ready to go at all times, so I just am not able to quickly engage and fix things and then disengage. That's life, no worries.

Reply:
Hi, 
if you want documentation on "how to edit/contribute to an existent Pull request", see https://github.com/ontoportal/administration/issues/11#issuecomment-1352837541

Reply:
thanks!  ok think I did it. resolve (and go ahead and merge, yes?) if you agree